blog/_posts/2018-07-09-introducing-TeamBarcelonaDuo.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Introducing Team Barcelona Duo
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/introducing-TeamBarcelonaDuo
```

Comment:
Permalink should be same as the file name. You can check the old blog posts for the example.